### PR TITLE
Update changelog for v3.3.0

### PR DESCRIPTION
**Pull request by Jaifroid**
This is still work in progress.

### CODE REVIEW

File: CHANGELOG.md
Comment (mossroy) [inline CHANGELOG.md:12]: It would be better to give a more specific explanation here.
With the line above, one could conclude that we now default to SW mode in Firefox extensions (which is wrong)

```suggestion
* NEW: App now defaults to Service Worker mode if accessed directly as a PWA (does not apply to browser extensions)
```

Comment (Jaifroid) [reply CHANGELOG.md]: Thanks, I committed that change.

File: CHANGELOG.md
Comment (mossroy) [inline CHANGELOG.md:14]: I suppose it's only about jQuery mode? If so, it would be better to write it here

**Comment by Jaifroid** [comment]
I've added changelog items for #804 , because I think it's ready. Obviously I won't merge this changelog until after merging #804 if it is indeed ready. Let me know if there are in principle any further items you'd like to add to this changelog, @mossroy .

**Comment by Jaifroid** [comment]
As far as I'm concerned, this changelog is now complete.

**Review by mossroy — APPROVED** [review]
Looks good!
You can merge

**Pull request merged**
